pkg/source/reconciler/mtsource/controller.go

```diff
@@ -40,6 +40,8 @@ import (
 type envConfig struct {
 	SchedulerRefreshPeriod int64 `envconfig:"AUTOSCALER_REFRESH_PERIOD" required:"true"`
 	PodCapacity            int32 `envconfig:"POD_CAPACITY" required:"true"`
+	NumZones               int32 `envconfig:"SCHEDULE_SPREAD_ZONES" required:"true"`
```

Comment:
Is this strictly needed? I assume you can compute the number of zones by getting all nodes, right?